examples/seismic/elastic/wavesolver.py

```diff
@@ -40,21 +48,37 @@ def forward(self, src=None, rec1=None, rec2=None, vp=None, vs=None, rho=None,
         Forward modelling function that creates the necessary
         data objects for running a forward modelling operator.
 
-        :param src: (Optional) Symbol with time series data for the injected source term
-        :param rec1: (Optional) Symbol to store interpolated (txx) receiver data
-        :param rec2: (Optional) Symbol to store interpolated (tzz) receiver data
-        :param vx: (Optional) Symbol to store the computed horizontal particle velocity
-        :param vz: (Optional) Symbol to store the computed vertical particle velocity
-        :param txx: (Optional) Symbol to store the computed horizontal stress
-        :param tzz: (Optional) Symbol to store the computed vertical stress
-        :param txz: (Optional) Symbol to store the computed diagonal stresss
-        :param vp: (Optional) Symbol for the time-constant P-wave velocity (km/s)
-        :param vs: (Optional) Symbol for the time-constant S-wave velocity (km/s)
-        :param vs: (Optional) Symbol for the time-constant density (rho=1 for water)
-        :param save: Option to store the entire (unrolled) wavefield
+        Parameters
+        ----------
+        src : , optional
+            The time series data for the injected source term.
+        rec1 : , optional
+            Interpolated (txx) receiver data.
+        rec2 : ,optional
+            Interpolated (tzz) receiver data.
+        vx : TimeFunction, optional
+            The computed horizontal particle velocity.
+        vz : TimeFunction, optional
+            The computed vertical particle velocity.
+        txx : TimeFunction, optional
+            The computed horizontal stress.
+        tzz : TimeFunction, optional
+            The computed vertical stress.
+        txz : TimeFunction, optional
+            The computed diagonal stresss.
+        vp : TimeFunction, optional
+            The time-constant P-wave velocity (km/s).
+        vs : TimeFunction, optional
+            The time-constant S-wave velocity (km/s).
+        vs : TimeFunction, optional
+            The time-constant density (rho=1 for water).
```

Comment:
rho

 `vp` and `vs` and `rho` are Function not TimeFunction

Reply:
fixed